### ISSUE TITLE
Install applications using Chocolatey

### ISSUE THREAD

**Issue by Pablohn26** [Fmstrat/winapps#29]
To have a more flexible way to install packages, (why not use Chocolatey](https://chocolatey.org/packages)?

Even ansible has a [module for it](https://docs.ansible.com/ansible/2.6/modules/win_chocolatey_module.html), so it would be to type the name of the package and ansible could install it using Chocolatey.

**Comment by Fmstrat** [comment]
I have never heard of this, but this sounds very promising! Any interest in working on the integration?

**Comment by Pablohn26** [comment]
yes, I am interested but I do not have enough time right now to address it. You can go ahead if you want or when I have time I can submit some PR

It is basically to [follow this documentation](https://docs.ansible.com/ansible/latest/user_guide/windows_setup.html) and then is done because [Ansible module](https://docs.ansible.com/ansible/2.6/modules/win_chocolatey_module.html) says: If Chocolatey is missing from the system, the module will install it.

**Comment by fbartels** [comment]
Another popular package manager on windows is https://scoop.sh/

**Comment by Tiebe** [comment]
> Another popular package manager on windows is https://scoop.sh/
With some experimenting I found that these 2 commands install Scoop without any confirmation:

Set-ExecutionPolicy RemoteSigned -scope CurrentUser -force
Invoke-Expression (New-Object System.Net.WebClient).DownloadString('https://get.scoop.sh')

**Comment by shoeper** [comment]
When I tried chocolatey before, it was quite unreliable. Not sure whether that changed in the meantime.

**Comment by DeltaWhy** [comment]
Windows recently got its own official package manager:
https://docs.microsoft.com/en-us/windows/package-manager/winget/
